frontend/src/lib/components/Support/SupportForm.tsx

```diff
@@ -136,15 +135,56 @@ export function SupportForm(): JSX.Element | null {
                         </>
                     </LemonBanner>
                 )}
-            <LemonField name="severity_level" label="What is the severity of this issue?">
+            <Field name="target_area" label="What area does this best relate to?">
+                {({ value, onChange }) => (
+                    <LemonButtonWithDropdown
+                        fullWidth
+                        type="secondary"
+                        sideIcon={undefined}
+                        dropdown={{
+                            className: 'Popover__TargetAreaField',
+                            placement: 'bottom-start',
+                            overlay: (
+                                <div className="TargetAreaField__dropdown">
+                                    {TARGET_AREA_TO_NAME.map(({ label, values }, i) =>
+                                        Object.keys(values).length != 0 ? (
+                                            <div key={i}>
+                                                {i !== 0 && <LemonDivider />}
+                                                <h5>{label}</h5>
+                                                {Object.entries(values).map(([key, _value]) => (
+                                                    <LemonButton
+                                                        key={key}
+                                                        onClick={() => {
+                                                            onChange(key)
+                                                        }}
+                                                        active={key == value}
+                                                        fullWidth
+                                                    >
+                                                        {_value}
+                                                    </LemonButton>
```

Comment:
i guess this is the equivalent of the lemon select previously? 

`'data-attr': `support-form-target-area-${key}`,`

it's still missing this data-attr

Reply:
why not use https://storybook.posthog.net/?path=/story/lemon-ui-lemon-select--sectioned-options

?